llvm/test/tools/yaml2obj/GOFF/invalid-record.yaml

```diff
@@ -0,0 +1,7 @@
+# RUN: not yaml2obj %s 2>&1 | FileCheck %s
+
+# CHECK: YAML:{{[0-9]*}}:{{[0-9]*}}: error: unknown record type name 'Foo'
```

Comment:
Note, we usually omit the part before `error:`.

Just use `CHECK: error: xxx` (`k` typo below?)